Mixpanel/MixpanelPeople.m

```diff
@@ -132,6 +132,20 @@ - (void)removePushDeviceToken
     [self addPeopleRecordToQueueWithAction:@"$unset" andProperties:properties];
 }
 
+- (void)removeSinglePushDeviceToken:(NSData *)deviceToken
+{
+    const unsigned char *buffer = (const unsigned char *)deviceToken.bytes;
+    if (!buffer) {
+        return;
+    }
+    NSMutableString *hex = [NSMutableString stringWithCapacity:(deviceToken.length * 2)];
+    for (NSUInteger i = 0; i < deviceToken.length; i++) {
+        [hex appendString:[NSString stringWithFormat:@"%02lx", (unsigned long)buffer[i]]];
+    }
+    NSDictionary *properties = @{@"$ios_devices": [NSString stringWithString:hex]};
+    [self addPeopleRecordToQueueWithAction:@"$remove" andProperties:properties];
+}
+
```

Comment:
Seeming that this is an exact duplicate of the contents of addPushDeviceToken aside from the last line, I would create a separate method that converts the NSData to NSString and reuse that